identity_service/repository/mock_token_repository.py

```diff
@@ -61,21 +74,56 @@ async def get_refresh_token(self, user_id: str) -> str:
             No refresh token found for provided user_id
 
         """
-        result = self._tokens.get(user_id)
+        result = None
+
+        for user in self._tokens.values():
```

Comment:
maybe use next(user[session_id] for user in self._tokens.values() if session_id in user) and try except StopIteration instead

Reply:
fixed